lib/ReactViews/BottomDock/ChartDisclaimer.tsx

```diff
@@ -18,14 +19,18 @@ interface ChartDisclaimerProps {
 const ChartDisclaimer: React.FC<ChartDisclaimerProps> = ({ terria }) => {
   const chartView = new ChartView(terria);
 
-  const chartableItemsWithDisclaimers: (Chartable &
-    DiscretelyTimeVaryingMixin.DiscretelyTimeVaryingMixin)[] = chartView.chartableItems
-    .filter(DiscretelyTimeVaryingMixin.isMixedInto)
-    .filter(item => item.chartDisclaimer);
-  if (chartableItemsWithDisclaimers.length === 0) return null;
-  const uniqueChartDisclaimers = [
-    ...new Set(chartableItemsWithDisclaimers.map(i => i.chartDisclaimer))
```

Comment:
I rewrote this bit to fix a type error I was getting after removing `forceLoadChartItems` from the Mixin.